src/nls/root/strings.js

```diff
@@ -607,6 +610,13 @@ define(function (require, exports, module) {
                 VECTOR: "Vector",
                 SMARTOBJECT: "Smart Object",
                 GROUP: "Group"
+            },
+            MODIFIERS: {
```

Comment:
What better place than here, what better time than now?
To localize all modifier strings!